test/extended/etcd/invariants.go

```diff
@@ -20,11 +26,28 @@ var _ = g.Describe("[sig-etcd] etcd", func() {
 		masterNodeLabelSelectorString := "node-role.kubernetes.io/master"
 		masterNodeList, err := oc.KubeClient().CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{LabelSelector: masterNodeLabelSelectorString})
 		o.Expect(err).ToNot(o.HaveOccurred())
+
+		ipFamily := getCurrentNetworkTopology(oc)
```

Comment:
otherwise, it fails with:
```
incorrect number of voting members found in openshift-etcd/etcd-endpoints, expected it to match the number of master nodes = 6, members from the cm = [192.168.111.20 192.168.111.21 192.168.111.22], 

master nodes = [192.168.111.20 192.168.111.21 192.168.111.22 fd2e:6f44:5dd8:c956::14 fd2e:6f44:5dd8:c956::15 fd2e:6f44:5dd8:c956::16]
```

Reply:
I feel like this has a potential to live in our utils in the long run.